picky/src/ssh/public_key.rs

```diff
@@ -52,20 +53,9 @@ impl FromStr for SshPublicKey {
 mod tests {
     use super::*;
 
-    use digest::Digest;
+    use crate::test_files;
     use num_bigint_dig::BigUint;
-
```

Comment:
I don't think that a public key **string** comparison via MD5 is better than simple string comparison... 😨

Reply:
Sounds about right! Just to be safe, do you remember why? @RRRadicalEdward 
https://github.com/Devolutions/picky-rs/blob/308c09ae786d0166f1bcbe6abcba576d324426b0/picky/src/ssh/public_key.rs#L58
https://github.com/Devolutions/picky-rs/blob/308c09ae786d0166f1bcbe6abcba576d324426b0/picky/src/ssh/public_key.rs#L150

Reply:
Seems like leftover code. I might have been valid at some point, but now It doesn't seem to be valuable here